src/rollbar.php

```diff
@@ -246,24 +250,12 @@ protected function _report_exception($exc, $extra_data = null, $payload_data = n
 
         $data = $this->build_base_data();
 
-        // exception info
-        $message = 'unknown';
-        if (method_exists($exc, 'getMessage')) {
-            $message = $exc->getMessage();
-        }
-
         $data['body'] = array(
-            'trace' => array(
-                'frames' => $this->build_exception_frames($exc),
-                'exception' => array(
-                    'class' => get_class($exc),
-                    'message' => $message
-                )
-            )
+            'trace' => $this->build_exception_trace($exc, $extra_data),
         );
 
-        if ($extra_data !== null) {
-            $data['body']['trace']['extra'] = $extra_data;
+        if ($exc->getPrevious() instanceof Exception) {
+            $data['body']['trace_chain'] = $this->build_exception_trace_chain($exc);
```

Comment:
The Rollbar API expects there to be either a 'trace' OR a 'trace_chain', but not both. I'll push a commit to fix this.